controller/controller.go

```diff
@@ -267,13 +267,14 @@ func (c *Impl) processNextWorkItem() bool {
 	// resource to be synced.
 	if err = c.Reconciler.Reconcile(ctx, key); err != nil {
 		c.handleErr(err, key)
+		logger.Errorf("Reconcile failed. Time taken: %d ms.", time.Now().Sub(startTime)/time.Millisecond)
```

Comment:
note that if you just pass the duration with `%v` it will serialize with units e.g. `3s`